### PR TITLE
Type as set

### PR DESCRIPTION
**Pull request by gkellogg**
Allows `@type` to be defined with `@container: @set`. Most descriptions of behavior left to the API document.

### CODE REVIEW

**Comment by gkellogg** [comment]
See also w3c/json-ld-api#41.

File: index.html [hunk 1]
```diff
@@ -9008,6 +9029,9 @@ <h2>Context Definitions</h2>
     its value MUST be an <a>absolute IRI</a>, a <a>relative IRI</a>,
     or <a>null</a>.</p>
 
+  <p class="changed">If the <a>context definition</a> has an <code>@type</code> key,
+    its value MUST be a <a>dictionary</a> with the single member <code>@context</code> with value <code>@set</code>.</p>
```
Comment (davidlehn) [inline index.html]: Should be `@container`.  Probably should also be `with a value of` or `with the value`.

**Comment by davidlehn** [comment]
Adding an example sure causes a gigantic mess for an otherwise small change.

File: index.html (hunk 1 above)
Comment (davidlehn) [reply index.html]: Or maybe `<code>@container</code> set to <code>@set</code>` to align with the api text?

**Pull request merged**